src/cross_module_fn.h

```diff
@@ -27,6 +32,11 @@
  *     ts_cm_functions-><function name>
  */
 
+typedef struct JsonbParseState JsonbParseState;
+typedef struct Hypertable Hypertable;
+typedef struct Chunk Chunk;
+typedef struct CopyChunkState CopyChunkState;
+
 typedef struct CrossModuleFunctions
```

Comment:
Do we still need this `CrossModuleFunctions` in the future in the way it is implemented now? maybe we should rework it into something more simpler but still remain cross module separation if that makes sense

Reply:
CrossModule allows running a pure Apache-based version of TimescaleDB.